eng/pipelines/scripts/Copy-ApiScanEligible.ps1

```diff
@@ -0,0 +1,97 @@
+param(
+  [Parameter(Mandatory=$true)][string] $SourcePath,
+  [Parameter(Mandatory=$true)][string] $TargetPath,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+if ($null -ne $properties) {
+  Write-Host "Unexpected extra parameters: $properties."
+  exit 1
+}
+
+$copyCount = 0
+$skipCount = 0
+
+# Copy files from source to target directory while preserving the relative directory structure
+function Copy-File() {
+  param(
+    [System.IO.FileInfo] $FileInfo
+  )
+  Write-Host "Copying $($FileInfo.FullName)"
+  $relativePath = [System.IO.Path]::GetRelativePath($SourcePath, $FileInfo.FullName)
+  $fileTargetPath = Join-Path -Path $TargetPath -ChildPath $relativePath
+  $fileTargetDir = [System.IO.Path]::GetDirectoryName($fileTargetPath)
+  [System.IO.Directory]::CreateDirectory($fileTargetDir) | Out-Null
+  Copy-Item -Path $FileInfo.FullName -Destination $fileTargetPath
+  $script:copyCount++
+}
+
+# Check if the file is a ARM64 PE file
+function Test-PeArm64() {
+  param(
+    [System.IO.FileInfo] $FileInfo
+  )
+  $stream = [System.IO.File]::OpenRead($FileInfo.FullName)
```

Comment:
Didn't think I'd ever see reading a file's COFF header in powershell .... I don't have a better suggestion since Windows doesn't have *nix's `file` utility and a dependency on `dumpbin` seems excessive since it might not always be present. LGMT.

Reply:
Yeah, I tried looking for anything better than this, but there isn't any easy utility for it without pulling in some custom PowerShell modules.